### PR TITLE
Muti version

### PR DESCRIPTION
**Pull request by ttgenius**
debug prints will be deleted once merge the QSW branch
just in case i forget

### CODE REVIEW

**Comment by claudio525** [comment]
https://github.com/ucgmsim/slurm_gm_workflow/pull/161 needs to approved and merged before this can go through!

File: e2e_tests/E2ETests.py [hunk 1]
```diff
@@ -281,29 +306,40 @@ def check_install(self):
...
+        submit_cmd = (
+            "python3 {} {} {} --sleep_time 2 --config {} "
+            "--no_merge_ts --no_clean_up".format(
+                os.path.join(
+                    os.path.dirname(os.path.abspath(__file__)),
+                    "../scripts/cybershake/auto_submit.py",
+                ),
+                self.stage_dir,
+                user,
+                os.path.join(self.stage_dir, os.path.basename(self.config_dict[self.cf_cybershake_config_key]))
```
[the remark this answers is not in the document]
Comment (ttgenius) [reply e2e_tests/E2ETests.py]: This allows you to pass a cybershake_cofig file with a name other than "cybershake_config.json" inside e2e_config

File: shared_workflow/install_shared.py [hunk 2]
```diff
@@ -56,6 +56,7 @@ def install_simulation(
         vel_mod_params_dir = vel_mod_dir
 
     template_path = os.path.join(defaults.recipe_dir, "gmsim", version)
+    print("template_path",template_path)
```
Comment (claudio525) [inline shared_workflow/install_shared.py]: debug print, pls remove

File: e2e_tests/E2ETests.py [hunk 3]
```diff
@@ -284,7 +282,7 @@ def _check_true(self, check: bool, location: str, error_msg: str):
 
     def check_install(self):
         """Checks that all required templates exists, along with the yaml params """
-
+        print("sim_dirs", self.sim_dirs)
```
Comment (claudio525) [inline e2e_tests/E2ETests.py]: debug print, pls remove

File: e2e_tests/E2ETests.py [hunk 4]
```diff
@@ -248,8 +245,9 @@ def install(self):
                 self.stage_dir,
                 os.path.basename(self.config_dict[self.cf_fault_list_key]),
             ),
+            self.version
         )
-
+        print("cmd", cmd)
```
Comment (claudio525) [inline e2e_tests/E2ETests.py]: debug print, pls remove

File: e2e_tests/E2ETests.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (claudio525) [reply e2e_tests/E2ETests.py]: Ah yes, pretty sure we even talked about this, I just forgot..

**Pull request merged**